### ISSUE TITLE
Broken example in README

### ISSUE THREAD

**Issue by manuzhang** [databricks/koalas#120]
`spark.from_pandas` in the following example should be `koalas.from_pandas` and the import `from databricks import koalas` is also missing

```python
import pandas as pd
pdf = pd.DataFrame({'x':range(3), 'y':['a','b','b'], 'z':['a','b','b']})

df = spark.from_pandas(pdf)

# Rename the columns
df.columns = ['x', 'y', 'z1']

# Do some operations in place:
df['x2'] = df.x * df.x
```

**Comment by rxin** [comment]
We are definitely in the middle of a refactoring to change the API https://github.com/databricks/koalas/issues/90

Can you submit a pull request to fix the README?

**Comment by manuzhang** [comment]
Sure. Just realized the example does work if I add `import databricks.koalas` and create a `SparkSession` beforehand. Encapsulation is easier to wrap my mind around.

**Comment by manuzhang** [comment]
Opened https://github.com/databricks/koalas/pull/126. Please help review.

**Comment by manuzhang** [comment]
Fixed in #126, thanks for review
